beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -350,6 +350,10 @@ func (vs *Server) ProposeBeaconBlock(ctx context.Context, req *ethpb.GenericSign
 		}
 		sidecars := make([]*ethpb.BlobSidecar, len(scs))
 		for i, sc := range scs {
+			log.WithFields(logrus.Fields{
+				"blockRoot": hex.EncodeToString(sc.Message.BlockRoot),
+				"indices":   i,
```

Comment:
`index` would make more sense as a key name than the plural. I would also rather use the `.Index` attribute than infer the index from the position in the list.